magma/primitives/mux.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, height: int, T: Type):
             mux.I.sel @= io.S
         out_ts = mux.O.ts
         if issubclass(T, MagmaProtocol):
-            out = T._from_magma_value_(T._to_magma_().unflatten(out_ts))
+            out = T._qualify_magma_(Direction.Out)._from_magma_value_(T._to_magma_().unflatten(out_ts))
```

Comment:
```suggestion
            out = T._to_magma_().unflatten(out_ts)
            out = T._qualify_magma_(Direction.Out)._from_magma_value_(out)
```

Need to shorten line length for pycodestyle